docs/API_documentation/Overview/Getting_access.md

```diff
@@ -55,11 +55,11 @@ curl -i -X - `GET` "http://#HOSTNAME/api/coiotedm/v3/devices" -H "accept: applic
   
 **Token expiration time**
 
-The expiration time of the token is determined by the administrator and remains consistent for all user types. The token expires after 5 minutes.
+The expiration time of the token is determined by the administrator and remains consistent for all user types. Typically, the token expires after around 5 minutes.
 
 ## Permissions
 
-Each endpoint has a separate permission. You can find information about permissions required for accessing a specific endpoint in the Guides section.
+Each endpoint has a separate permission. 
 
 Developer and Business accounts have permissions for the access to the following set of API endpoints:
 
```

Comment:
This list should be more highlighted like e.x. `GET cachedDataModels/{deviceId}`